src/app/tl-header/tl-header.component.scss

```diff
@@ -403,6 +403,23 @@
     left: 0;
     z-index: 1100;
     transition: width 0.5s ease-in-out, opacity 0.5s ease-in-out;
+    .flex-fill.align-self-stretch {
```

Comment:
Please target identity based classes instead of functional classes.

Reply:
Removed the boostrap based functional classes and added the div section instead.